examples/lsm303_accel_inclinometer.py

```diff
@@ -0,0 +1,38 @@
+""" Display inclination data five times per second """
+
+import time
+from math import atan2, degrees
+import board
+import busio
+import adafruit_lsm303_accel
+
+
+i2c = busio.I2C(board.SCL, board.SDA)
+sensor = adafruit_lsm303_accel.LSM303_Accel(i2c)
+
+
+def vector_2_degrees(x, y):
+    radians = atan2(y, x)
+    degrees_calc = degrees(radians)
+    if degrees_calc < 0:
+        degrees_calc = 360 + degrees_calc
+    return degrees_calc
+
+
+def get_inclination(_sensor):
+    return get_inclination_respect_x(_sensor), get_inclination_respect_y(_sensor)
```

Comment:
With this approach you read the sensor twice. It probably happens so fast it's no big deal. But you could get rid of the two `get_inclination_respect` functions and have this one be something like:
```python
x, y, z = _sensor.acceleration
return vector_2_degrees(x, z), vector_2_degrees(y, z)
```
and then both inclination computations would be done from the same sensor reading.

Reply:
Thank you for pointing this out. Like you say it's so fast it usually doesn't matter but I have had the pleasure of debugging weirdness caused by making multiple reads like this getting different results on with other sensors, definitely should be storing it to a variable to reference from there instead.  I will get this changed tonight as well.